### PR TITLE
Support Swift 5.1 and add new features

### PR DESCRIPTION
**Pull request by X901**
- **Add Dark Mode support**
Now by default it suppourt Dark mode , depend on the iOS settings

-  **Add: RTL support for language like Arabic**
I add new property  `positionType` 
user can choose .left or .right 

- **Add: Swift 5.1 support**

- **Fix: Constraints issue**
there was issue with Constraints I fixed it

### CODE REVIEW

**Comment by X901** [comment]
I don't get it why it was failed ?
it was testing in iOS 10 

```
Testing failed:
	Type 'UITableView' has no member 'automaticDimension'
	Type 'UIColor?' has no member 'label'
	Type 'UIColor?' has no member 'label'
	Type 'UIColor?' has no member 'label'
	Type 'UIColor?' has no member 'systemGroupedBackground'
	Type 'UIColor?' has no member 'secondarySystemGroupedBackground'
	Type 'UIColor?' has no member 'systemGroupedBackground'
	Type 'UIColor?' has no member 'label'
```
What I missing here ? 

This my frist time I collaborated on open source

File: Source/FAQView.swift [hunk 1]
```diff
@@ -462,27 +513,44 @@ class FAQViewCell: UITableViewCell {
     self.answerTextView.textColor = configuration.answerTextColor
     self.questionLabel.font = configuration.questionTextFont
     self.answerTextView.font = configuration.answerTextFont
+    
+    if let positionType = configuration.positionType {
+    setupTextAlignment(position: positionType)
+    addConstraintsDependOnPositionType(position: positionType)
```
Comment (mukeshthawani) [inline Source/FAQView.swift]: Can you please format the code to match existing indentation?

**Review by mukeshthawani — CHANGES_REQUESTED** [review]
Thanks for creating the PR! Looks like there are some formatting issues, it should be resolved once you re-indent the changes. I'll review it again after some time.

**Comment by mukeshthawani** [comment]
I've updated Travis config, now the build won't fail. Can you rebase your branch with the master?

File: Source/FAQView.swift (hunk 1 above)
Comment (X901) [reply Source/FAQView.swift]: Is what I did on last commit is what you asked for ? , or I understand it wrong ?

**Pull request merged**